docs/helpers/snippets/decodeJwt.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+import type { CredentialResponse } from "vue3-google-signin";
+import { GoogleSignIn } from "vue3-google-signin";
+import { decodeJWTToken } from "vue3-google-signin/utils";
+
+const handleSignInSuccess = (response: CredentialResponse) => {
+  const { credential } = response;
+  const decodedCredential = decodeJWTToken(credential);
+  console.log("Acess Token", decodedCredential);
```

Comment:
`console.log("Credentials", decodedCredential);`

Reply:
Doh! That's what you get when using copy/paste too quickly 👅
Thanks for the doc bug. I'll fix it quickly